src/python/ddapp/tasks/robottasks.py

```diff
@@ -706,6 +708,15 @@ def run(self):
         side = self.properties.getPropertyEnumValue('Side').lower()
         self.getHandDriver(side).sendCustom(self.properties.getProperty('Amount'), 100, 100, self.properties.getProperty('Mode'))
 
+        if self.properties.getProperty('Check status'):
+            responseChannel = 'GRASPING_STATE'
+            responseMessageClass = lcmdrc.boolean_t
+            grasping_state = lcmUtils.MessageResponseHelper(responseChannel, responseMessageClass).waitForResponse(timeout=7000)
+            if grasping_state is not None and grasping_state.data == 1:
+                self.statusMessage = "Grasping successful"
+            else:
+                self.fail("No object in hand")
+
```

Comment:
You might consider putting this code into a new task called WaitForGraspingState or something like that.  Then you can replace these lines with:

   WaitForGraspingState().run()

That will be useful in case people want to parameterize the WaitForGraspingState task in the future.